extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -257,6 +260,30 @@ export class AmpAnalytics extends AMP.BaseElement {
   }
 
   /**
+   * If requestsFrameUrl (and optionally dataHash as well) are specified in
+   * config, check whether third-party iframe already exists, and if not,
+   * create it.
+   * @param {!JSONType} config
+   * @private
+   */
+  handleRequestsFrameUrl_(config) {
+    if (config['requestsFrameUrl'] && config['dataHash']) {
+      if (!config['dataHash'].startsWith('#')) {
```

Comment:
Please use the `startsWith` [helper](https://github.com/ampproject/amphtml/blob/9041fc1332aa5c5d8695199afff1fc827bcfd799/src/string.js#L46-L57).